modules/openapi-generator/src/main/java/org/openapitools/codegen/CodegenProperty.java

```diff
@@ -704,16 +704,85 @@ public boolean equals(Object o) {
     @Override
     public int hashCode() {
 
-        return Objects.hash(openApiType, baseName, complexType, getter, setter, description, dataType,
-                datatypeWithEnum, dataFormat, name, min, max, defaultValue, defaultValueWithParam, baseType,
-                containerType, title, unescapedDescription, maxLength, minLength, pattern, example, jsonSchema,
-                minimum, maximum, exclusiveMinimum, exclusiveMaximum, hasMore, required, secondaryParam,
-                hasMoreNonReadOnly, isPrimitiveType, isModel, isContainer, isString, isNumeric, isInteger,
-                isLong, isNumber, isFloat, isDouble, isByteArray, isBinary, isFile, isBoolean, isDate, isDateTime,
-                isUuid, isUri, isEmail, isFreeFormObject, isListContainer, isMapContainer, isModelContainer, isEnum,
-                isReadOnly, isWriteOnly, isNullable, isSelfReference, isCircularReference, _enum, allowableValues,
-                items, mostInnerItems, vendorExtensions, hasValidation, isInherited, discriminatorValue, nameInCamelCase,
-                nameInSnakeCase, enumName, maxItems, minItems, isXmlAttribute, xmlPrefix, xmlName, xmlNamespace,
-                isXmlWrapped, multipleOf);
+    return Objects.hash(
```

Comment:
I've noticed multiple times now that this list continuously causes conflicts, when separated by spaces only. Changing it to line breaks again.